src/Common/tests/System/Resources/Binary/TestData.cs

```diff
@@ -0,0 +1,165 @@
+                ["sbyte"] = (sbyte)-3,
+                ["short"] = (short) 31000,
+                ["ushort"] = (ushort) 61000,
+                ["long"] = 10000000000,
+                ["ulong"] = ulong.MaxValue,
+                ["float"] = 3.14f,
+                ["double"] = 3.14159,
+                ["decimal"] = 3.141596536897931m
+            };
+
+        public static IReadOnlyDictionary<string, object> BinaryFormatted
+        {
+            get => PlatformDetection.IsDrawingSupported ?
+                  BinaryFormattedDrawing : BinaryFormattedWithoutDrawing;
+        }
+        public static Dictionary<string, object> BinaryFormattedWithoutDrawing { get; } =
+            new Dictionary<string, object>()
+            {
+                ["enum_bin"] = DayOfWeek.Friday,
+                ["point_bin"] = new Point(4, 8)
+            };
+
+        public static IReadOnlyDictionary<string, object> BinaryFormattedDrawing
+        {
+            get => new Dictionary<string, object>(BinaryFormattedWithoutDrawing)
+            {
+                ["bitmap_bin"] = new Bitmap(Path.Combine("bitmaps", "almogaver24bits.bmp")),
+                ["font_bin"] = SystemFonts.DefaultFont
+            };
+        }
+
+        public static IReadOnlyDictionary<string, object> ByteArrayConverter
+        {
+            // ImageConverter is part of System.Windows.Extensions.
+            get => PlatformDetection.IsDrawingSupported && PlatformDetection.IsWindows ?
+                ByteArrayConverterDrawing : ByteArrayConverterWithoutDrawing;
+        }
+
+        public static Dictionary<string, object> ByteArrayConverterWithoutDrawing { get; } =
+            new Dictionary<string, object>()
+            {
+                ["myResourceType_bytes"] = new MyResourceType(new byte[] { 0xAB, 0xCD, 0xEF, 0x01, 0x23, 0x45, 0x67, 0x89 })
+            };
+
+        public static IReadOnlyDictionary<string, object> ByteArrayConverterDrawing
+        {
+            get => new Dictionary<string, object>(ByteArrayConverterWithoutDrawing)
+            {
+                ["bitmap_bytes"] = new Bitmap(Path.Combine("bitmaps", "almogaver24bits.bmp")),
+                ["icon_bytes"] = new Icon(Path.Combine("bitmaps", "32x32_one_entry_4bit.ico"))
+            };
+        }
+
+        public static IReadOnlyDictionary<string, object> StringConverter
+        {
+            // ImageFormatConverter is part of System.Windows.Extensions.
+            get => PlatformDetection.IsDrawingSupported && PlatformDetection.IsWindows ?
+                StringConverterDrawing : StringConverterWithoutDrawing;
+        }
+
+        public static Dictionary<string, object> StringConverterWithoutDrawing { get; } =
+            new Dictionary<string, object>()
+            {
+                ["color_string"] = Color.AliceBlue,
+                ["point_string"] = new Point(2, 6),
+                ["rect_string"] = new Rectangle(3, 6, 10, 20),
+                ["size_string"] = new Size(4, 8),
```

Comment:
For some reason this converter isn't working in CI.  It's falling back to default converter which fails
```
System.ArgumentException : 4, 8 is not a valid value for Int32.
Parameter name: value
---- System.FormatException : Input string was not in a correct format
```

Reply:
I think this is caused by not using CultureInfo that was used when writing the .resources.  CI likely has a different ListSeparator since it is running on ES.

Reply:
Need to use ConvertFromInvariantString https://referencesource.microsoft.com/System/A.html#a4f6cd8112ba29f6